spec/api_v1.yaml

```diff
@@ -0,0 +1,368 @@
+openapi: "3.0.2"
```

Comment:
Are there any recommendations/suggestions on how to keep this file up to date with the code?

Reply:
I'm not sure. I was planning to make this a part of the code review process.

If we want to make this automated, maybe we can create a pytest plugin/hook that records all of API calls in the unit tests and verifies there is a corresponding entry in the OpenAPI spec for that route, method, and status code. It could be something to explore if we want to add this to the rest of our projects.